src/pytest_plugins/est_help/est_help.py

```diff
@@ -0,0 +1,67 @@
+"""
+A small pytest plugin that shows the a concise help string that only contains
+the options defined by the plugins defined in execution-spec-tests (est).
+"""
+
+import argparse
+
+import pytest
+
+
+def pytest_addoption(parser):
+    """
+    Adds command-line options to pytest.
+    """
+    help_group = parser.getgroup(
+        "est_help", "Arguments related to getting help for execution-spec-tests:"
```

Comment:
```suggestion
        "est_help", "Arguments related to getting help for execution-spec-tests"
```
Otherwise the colon is printed twice in the help output.

Reply:
Well spotted! Spencer fixed it in his PR.